packages/entity-browser/src/modules/entityBrowser/reducer.js

```diff
@@ -14,14 +14,24 @@ const clearRecordStore = state => ({
   recordStore: {}
 })
 
+const setSearchTerm = (state, {payload}) => {
+  const term = payload.searchTerm
```

Comment:
I don't think the name should be saved in the redux state. The name only can be used to set/change the search term directly in the reducer. There is no need for the name in the state.